collect_app/src/test/java/org/odk/collect/android/projects/ManualProjectCreatorDialogTest.kt

```diff
@@ -112,7 +112,7 @@ class ManualProjectCreatorDialogTest {
             onView(withHint(R.string.password)).perform(replaceText("1234"))
 
             onView(withText(R.string.add)).perform(click())
-            verify(projectCreator).createNewProject("{\"general\":{\"server_url\":\"https:\\/\\/my-server.com\",\"username\":\"adam\",\"password\":\"1234\"},\"admin\":{}}")
+            verify(projectCreator).createNewProject("{\"general\":{\"server_url\":\"https:\\/\\/my-server.com\",\"username\":\"adam\",\"password\":\"1234\"},\"admin\":{},\"project\":{}}")
```

Comment:
Why does there need to be a `project` object in the JSON here?

Reply:
Because now it's a part of the json generated by our app. And like `admin` section which also in this case is empty it exists there.

Reply:
Right I was just checking it doesn't **need** to be there (that we still support QR codes with `project`) but I've double-checked and it looks like we're ok!